inspire/dojson/common/base.py

```diff
@@ -39,6 +39,12 @@ def control_number(self, key, value):
     return value[0]
 
 
+@hep2marc.over('001', 'control_number')
+def control_number2marc(self, key, value):
+    """Record Identifier."""
+    return [value]
```

Comment:
I guess the question here is if ```create_record``` should put the controlfields inside of a list as it is currently doing. (and avoid `return value[0]` doing just `return value`